src/coreclr/src/ilasm/grammar_after.cpp

```diff
@@ -1542,18 +1534,17 @@ char* AsmParse::fillBuff(__in_opt __nullterminated char* pos)
     }
     else
     {
-        if(((penv->curPos[0]&0xFF)==0xEF)&&((penv->curPos[1]&0xFF)==0xBB)&&((penv->curPos[2]&0xFF)==0xBF))
+        if((penv->curPos[0] & 0xFF) == 0xEF && (penv->curPos[1] & 0xFF) == 0xBB && (penv->curPos[2] & 0xFF) == 0xBF)
         {
             g_uCodePage = CP_UTF8;
             penv->curPos += 3;
             if(assem->m_fReportProgress) printf("Source file is UTF-8\n\n");
-            penv->pfn_nextchar = nextcharU;
         }
         else
         {
-            if(assem->m_fReportProgress) printf("Source file is ANSI\n\n");
-            penv->pfn_nextchar = nextcharA;
+            if(assem->m_fReportProgress) printf("Source file is ANSI which is not supported. Treating as UTF-8\n\n");
```

Comment:
I would just print `"Source file is UTF-8\n\n` here (same as in the other path). Pretend that the ANSI encoding support in ilasm never existed.